src/Robo/Commands/Setup/SettingsCommand.php

```diff
@@ -217,15 +217,16 @@ public function gitHooks() {
   protected function installGitHook($hook) {
     if ($this->getConfigValue('git.hooks.' . $hook)) {
       $this->say("Installing $hook git hook...");
-      $source = $this->getConfigValue('git.hooks.' . $hook) . "/$hook";
-      $dest = $this->getConfigValue('repo.root') . "/.git/hooks/$hook";
+      $bltHook = $this->getConfigValue('git.hooks.' . $hook) . "/$hook";
+      $projectHookDirectory = $this->getConfigValue('repo.root') . "/.git/hooks";
```

Comment:
Non-class property variables should be camel case.

Reply:
noted!